recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,8 @@
+# So far, we just need a container to be inherited
+inherit ${@oe.utils.ifelse(d.getVar('UBOOT_PROVIDES_BOOT_CONTAINER') == '1', 'imx-boot-container', '')}
+
+# Location known to imx-boot component, where U-Boot artifacts
+# should be additionally deployed.
+# See below note above do_deploy:append:mx8m for the purpose of
+# this delopyment location
+BOOT_TOOLS = "imx-boot-tools"
```

Comment:
Hi Andrey

I wonder if we should move the setting of BOOT_TOOLS to imx-base for all recipes using it.
The variable is used to coordinate the location of artifact providers (imx-atf imx-sc-firmware u-boot-imx) to its consumers (u-boot-fslc, imx-boot  and now u-boot).
I.e. all of these recipes must agree on the same location or the build will fail.

Max

Reply:
Agree, this would be the right thing to do. I can try this out and either push it to this PR or make a separate one.

Thanks for pointing it out! 👍🏻